tests/test_trino_utils.py

```diff
@@ -9,45 +9,57 @@
 
 
 def test_trino_connection_params_from_url_with_auth():
-    url = "trino://user:password@host:80/catalog/schema"
-    params = trino_connection_params_from_url(url)
-
-    auth = params.pop("auth")
-    assert auth._username == "user"
-    assert auth._password == "password"
-
-    assert params == {
-        "http_scheme": "http",
-        "user": "user",
-        "host": "host",
-        "port": 80,
-        "catalog": "catalog",
-        "schema": "schema",
-    }
+    urls = [
+        "trino://user:password@host:80/catalog/schema",
+        "presto://user:password@host:80/catalog/schema",
+    ]
```

Comment:
Thanks for confirming. I'll do this, because I want to know for my own curiosity, and not having used pytest before.

(Incidentally, the "loop through test cases" approach is [idiomatic Go](https://golang.org/doc/code#Testing), which is my defence here :wink:)

Reply:
What I was missing yesterday was that you must pass a variable set with `parameterize` into the test function itself.